devnet-sdk/devstack/dsl/doc.go

```diff
@@ -0,0 +1,10 @@
+/*
+Package dsl provides DSL (domain specific language) to interact with a devstack system.
+
+Each component in the devstack has a DSL wrapper.
+The wrapper itself does not have any state, and may be recreated or shallow-copied.
+
+Each DSL wrapper provides an Escape method, in case the DSL is not sufficient for a given use-case.
+The Escape method usage should be kept to a minimum, for readability and maintainability.
```

Comment:
I think we should be stronger on this.  Once we have the migration to DSL well under way we should remove the `Escape()` method so that as people write new tests they write the DSL to support it and the DSL keeps evolving.  Otherwise the DSL will never be expanded to new features because people keep using the escape process. It's a useful function to migrate more incrementally, but super critical to get rid of as soon as we (sensibly) can.

```suggestion
The Escape method is a temporary compromise to allow more incremental development of and
migration to the DSL. It should be avoided whenever possible and will be removed in the future.
```